interface/src/avatar/MyAvatar.cpp

```diff
@@ -1480,7 +1480,16 @@ void MyAvatar::updateMotors() {
     if (_motionBehaviors & AVATAR_MOTION_ACTION_MOTOR_ENABLED) {
         glm::quat actionOrientation;
         if (qApp->isHMDMode() && _hmdRollControlEnabled) {
+#ifdef USE_SIMPLE_ROLL_CONTROL
+            glm::quat avatarOrientation = getOrientation();
+            glm::vec3 avatarUp = avatarOrientation * IDENTITY_UP;
+            glm::vec3 headForward = getHeadOrientation() * IDENTITY_FORWARD;
+            float pitchAngle = asinf(glm::clamp(glm::dot(headForward, avatarUp), -1.0f, 1.0f));
+            glm::quat prePitch = glm::angleAxis(pitchAngle, IDENTITY_RIGHT);
+            actionOrientation = avatarOrientation * prePitch;
```

Comment:
We don't need any history to compute the `actionOrientation`.  It is a function of the current head and avatar orientations.